docproc/src/test/java/com/yahoo/docproc/jdisc/DocumentProcessingHandlerTransformingMessagesTestCase.java

```diff
@@ -202,14 +212,8 @@ public Progress process(Processing processing) {
                     //nada
                 } else if ("doc:nodocstatus:update:to:nothing".equals(id)) {
                     it.remove();
-                } else if ("userdoc:12345:6789:multiop:nodocstatus:keep:this".equals(id)) {
+                } else if ("id:12345:6789:multiop:nodocstatus:keep:this".equals(id)) {
```

Comment:
Should this have been `n=6789`? Doesn't seem to have broken anything, so maybe not tested at all...

Reply:
Good point, will check.